openslides/agenda/projector.py

```diff
@@ -10,11 +10,41 @@
 
 class ItemListSlide(ProjectorElement):
     """
-    Slide definitions for the agenda.
+    Slide definitions for Item model.
+
+    This is only for list slides.
+
+    Omit 'id' to get the whole agenda tree. Set 'id' to None to get a list
+    slide of all root items. Set 'id' to an integer to get a list slide of
+    the children of the metioned item.
```

Comment:
I am against a agenda slide which shows not only the children, but also the children of the children. I think this are to many items.

But if you want such a slide, then please also add this feature for each item. For example show item 5 with his children as tree. For that to work id and tree have to be different attributes in the projector_config.